pkg/network/server.go

```diff
@@ -1545,34 +1564,53 @@ func (s *Server) tryInitStateSync() {
 	if s.stateSync.IsInitialized() {
 		return
 	}
-
-	s.lock.RLock()
-	heights := make([]uint32, 0, len(s.peers))
-	for p := range s.peers {
-		if p.Handshaked() {
-			heights = append(heights, p.LastBlockIndex())
+	if !s.ServerConfig.NeoFSStateFetcherCfg.Enabled {
+		s.log.Info("state sync is initializing based on P2P network")
```

Comment:
I took more careful look (ref. https://github.com/nspcc-dev/neo-go/pull/3844/commits/0250b2f014b87abedbb0f8d9bf518785b9c9e75b#r2060638997), and it seems that StateSync module itself has its own initialization log (`try to sync state for the latest state synchronisation point`), so there's some redundancy in the node's logs. So I'd suggest to remove `state sync is initializing based on P2P network` and `state sync is initializing based on the NeoFS container` and extend the log of StateSync module with operation mode, this way it will be more clear.